KoreBuild-dotnet/build/_xunit-test.shade

```diff
@@ -51,6 +51,9 @@ projectFile=''
                         testArgs = " -parallel none";
                     }
 
+                    // Removing app domains to allow custom test discovery
+                    testArgs += " -noappdomain";
```

Comment:
Mvc has plenty of these. Try referencing a test which references two different versions of Newtonsoft.Json for e.g.